operator/main.go

```diff
@@ -236,6 +240,7 @@ func main() {
 		// if you are doing or is intended to do any operation such as perform cleanups
 		// after the manager stops then its usage might be unsafe.
 		// LeaderElectionReleaseOnCancel: true,
+		ClientDisableCacheFor: disableCacheFor,
```

Comment:
Maybe, let's add a comment that describes why we need this and link to the client issue https://github.com/kubernetes-sigs/controller-runtime/issues/550